<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kicad Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="KicadHelper.qrc">
    <normaloff>:/KicadHelper/kicad.png</normaloff>:/KicadHelper/kicad.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Project Folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leProject">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leProjectFolder">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabReplace">
       <attribute name="title">
        <string>Text Replace</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>PCB/Schematic:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leSchematic"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbFindSchematic">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Map CSV:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leMapCSV"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pbImportCSV">
          <property name="text">
           <string>Import CSV</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QTableWidget" name="lwWords">
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
          <column>
           <property name="text">
            <string>From</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>To</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QPushButton" name="pbTextReplace">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFootprint">
       <attribute name="title">
        <string>Footprint Checker</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QPushButton" name="pbCheckFP">
          <property name="text">
           <string>Check FootPrints in Schematics</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTableWidget" name="twLibraries">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <column>
           <property name="text">
            <string>Level</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbReloadLibraries">
          <property name="text">
           <string>Reload Libraries</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QListWidget" name="lwResults"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPartNumber">
       <attribute name="title">
        <string>PartNumber Added</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lePartsCSV"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QTableWidget" name="twParts">
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>FootPrint</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Digikey PN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>LCSC PN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>MPN PN</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Part Number CSV</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbImportParts">
          <property name="text">
           <string>Import Parts</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPushButton" name="pbSetPartsInSch">
          <property name="text">
           <string>Add Part Numbers to Schematics</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRename">
       <attribute name="title">
        <string>Rename Project</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="1">
         <widget class="QPushButton" name="pbRename">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbCopyFiles">
          <property name="text">
           <string>Copy Files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Old Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leOldName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>New Name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leNewName"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lwLogs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionOpen_Explorer"/>
    <addaction name="actionOpen_Logs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_Logs">
   <property name="text">
    <string>Open Logs</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionOpen_Explorer">
   <property name="text">
    <string>Open Explorer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="KicadHelper.qrc"/>
 </resources>
 <connections/>
</ui>
